shared/unlock-folders/device-list.desktop.js

```diff
@@ -29,23 +39,26 @@ export default class DeviceList extends Component<void, Props, void> {
   render () {
     return (
       <div style={{...globalStyles.flexBoxColumn, alignItems: 'center'}}>
-        <Text type='Body' style={styles.infoText}>Turn on one of your devices to unlock your folders:</Text>
-        <div style={styles.devicesContainer}>
-          {this.props.devices && this.props.devices.map(d => <DeviceRow key={d.deviceID} device={d} />)}
+        <Text type='Body' style={styles.infoText}>This computer and possibly others are unable to read some of your folders. To avoid losing data forever, please turn on one of the devices below:</Text>
+        <div style={{...styles.devicesContainer, height: this.props.paperKeysHidden ? 202 : 162}}>
+          {this.props.devices &&
+            this.props.devices.map(d => <DeviceRow key={d.deviceID} device={d} paperKeysHidden={this.props.paperKeysHidden} />)}
         </div>
-        <div style={styles.buttonsContainer}>
+        {!this.props.paperKeysHidden && <div style={styles.buttonsContainer}>
```

Comment:
until we can handle logging in with a paperkey lets hide this button. this whole design will go away soon so lets not spend time fixing this up further